### PR TITLE
TESTED and UPDATED for RPM Distro

### PR DESCRIPTION
**Pull request by ace139**
Tested on Fedora 23 x86_64 linux distribution (RPM based) and updated the documentation with install instructions.

### CODE REVIEW

File: 01.adapt/01.install/docs.md [hunk 1]
```diff
@@ -56,44 +83,43 @@ El Capitan:
 
 Xcode must be installed and Apple’s  license agreement accepted.
 ```
-xcode-select --install
+$ xcode-select --install
```
Comment (clusterfudge) [inline 01.adapt/01.install/docs.md:86]: So is the xcode dependency to bring in brew?

Comment (ace139) [inline 01.adapt/01.install/docs.md:86]: I haven't tested for Mac, I have access to only Debian and RPM. Temporarily maybe we can merge the PR until someone tests and send another PR for Mac OSX

**Comment by ace139** [comment]
@clusterfudge  Was there any problem with my PR? Please let me know, I will fix it.

**Comment by clusterfudge** [comment]
Looks good to me! Thanks.
